utils/debug/Debug.js

```diff
@@ -51,6 +55,22 @@ function Debug(view, datDebugTool, chartDivContainer) {
         }
     });
 
+    gui.add(state, 'debugCameraWindow').name('debug Camera').onChange(() => {
+        view.notifyChange(true);
+    });
+
+
+    let update = tileLayer.update;
+    gui.add(state, 'freeze').name('freeze update').onChange((newValue) => {
```

Comment:
The `freeze` option should be supported in production mode too.

Reply:
Pr updated
* freeze in production mode
* test with globe, planar and panorama